.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: Build git-stat
```

Comment:
@IlyaMoskva `git-stat` is not a correct project name

Reply:
Fixed